packages/stylex/src/StyleXTypes.d.ts

```diff
@@ -73,7 +73,11 @@ export type NestedCSSPropTypes = Partial<
   }>
 >;
 
-type UserAuthoredStyles = CSSPropertiesWithExtras | { [key: string]: unknown };
+type UserAuthoredStyles = {
+  [P in keyof CSSPropertiesWithExtras]?:
+    | CSSPropertiesWithExtras[P]
+    | { [key: string]: unknown };
+};
```

Comment:
This is incorrect.

Reply:
I don't understand why is wrong :( .Before this the type is `CSSPropertiesWithExtras`. I guess stylex want user using `CSSPropertiesWithExtras` as attr .